bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -148,9 +156,14 @@ spec:
               - -c
               - |
                 OVN_NODES_ARRAY=({{.OVN_NODES}})
+                OVN_NODE_IPS_ARRAY=({{.OVN_NODE_IPS}})
                 if [[ "$K8S_NODE" == "${OVN_NODES_ARRAY[0]}" ]]; then
+                  USEIPV6=
```

Comment:
Oh hey, this is neat! Can you just tell it to bind on `0.0.0.0` for the v4 case?